pom.xml

```diff
@@ -126,14 +130,6 @@
         </plugin>
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-assembly-plugin</artifactId>
-          <version>2.6</version>
-          <configuration>
-            <tarLongFileMode>gnu</tarLongFileMode>
```

Comment:
Should replace the tarLongFileMode with a `<assembly.tarLongFileMode>posix</assembly.tarLongFileMode>` in the `<properties>` section. (This should always have been posix, not gnu, anyway.)